### PR TITLE
[v-play] add venue description implementation

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/components-Sailfish/IconButton.qml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```
Comment (micuintus) [inline qml/components-Sailfish/IconButton.qml]: That ```IconButton``` abstraction is amazing, nice! I have also already thought about this a bit (when thinking about the Theme), and I head exactly something like this in mind ;). Of course, I haven't implemented it / tried it out. Very nice work!

One idea: What would you think about moving this ```iconBy()``` function to the Theme class? Maybe we could do sth. like 
```
Theme
{
...
  function icon(type, scale)
...
}
```

I see the following advantages with that approach:
1. We don't have provide a wrapper for the ```IconButton``` implementation.
2. We can use this class in other cases where we need icons with native look and feel.

**Comment by micuintus** [comment]
OMG, that already looks sooooo nice. I am really impressed!

File: qml/components-Sailfish/IconButton.qml (hunk 1 above)
Comment (jmastr) [inline qml/components-Sailfish/IconButton.qml]: Thank. I think it's a good idea to move it into the Themes, but shall the function then return an array with `[iconString, scalingFactor]` or what did you think of?

Comment (jmastr) [inline qml/components-Sailfish/IconButton.qml]: I'd suggest just to move the functions `iconBy(type)` into the according Themes and set the scaling factor from the outside, as it is now already.

Comment (micuintus) [inline qml/components-Sailfish/IconButton.qml]: My idea was that it actually returns the icon type that is used on the corresponding platform: A string on v-play and a (scaled) icon on Sailfish, which both can be directly assigned to the icon object of the ```IconButton```. But I just saw that we then need to export the ```pressed``` property as a parameter of this function, as well, right? Does v-play have sth. like this? I assume yes, and I assume it is named differently. ;)

Comment (jmastr) [inline qml/components-Sailfish/IconButton.qml]: They have something how the icon should look like AFTER it was pressed (what we actually need for the upcoming favourites feature), but not for the pressed state itself.

I am not perfectly clear, what you want to achieve. So I'd suggest to move the function as it is, so it is at the right place and to postpone further improvements. I have to think about the whole VenueDetails-design anyways.

Ok?

Comment (jmastr) [inline qml/components-Sailfish/IconButton.qml]: Oh, I was wrong, I think pressed is called `selected` in v-play...

Comment (micuintus) [inline qml/components-Sailfish/IconButton.qml]: OK. Lets do it scrummy-iteratively step by step.

**Comment by micuintus** [comment]
I'd see one alternative option: Only (re)implementing an ```IconButton``` in ```Silica4v-play``` with a ```selected``` property and use the ```icon``` function suggested above. 

Anyway, we currently use a hybrid between the approaches of #62 and #56.
Lets find out where the journey leads us to ;).

**Comment by jmastr** [comment]
But I would at least need an Silica implementation to "fake" the `type` property...

**Pull request merged**